src/BuildScriptGenerator/PhpVersions.cs

```diff
@@ -6,12 +6,12 @@ public static class PhpVersions
     {
         public const string PhpRuntimeBaseTag = "20220324.2";
         public const string PhpFpmRuntimeBaseTag = "20220325.1";
-        public const string ComposerVersion = "2.0.8";
+        public const string ComposerVersion = "2.3.4";
         public const string ComposerSetupSha384 = "906a84df04cea2aa72f40b5f787e49f22d4c2f19492ac310e8cba5b96ac8b64115ac402c8cd292b8a03482574915d1a8";
         public const string Composer19Version = "1.9.3";
         public const string Composer110Version = "1.10.19";
         public const string Composer20Version = "2.0.8";
-        public const string Composer22Version = "2.2.9";
+        public const string Composer22Version = "2.3.4";
```

Comment:
Should we create another variable here? It would make sense to create a new `Composer23Version`, similar to what we have for others

Reply:
That's a good point. I'll update `Composer22Version` to `Composer23Version` since I see no need in keeping `2.2.9` since removing it from here won't remove it from prod